vertica/README.md

```diff
@@ -0,0 +1,77 @@
+# Agent Check: Vertica
+
+## Overview
+
+This check monitors [Vertica][] through the Datadog Agent.
+
+## Setup
+
+### Installation
+
+The Vertica check is included in the [Datadog Agent][] package.
+
+The user used to connect to the database must be granted the [SYSMONITOR][monitor role] role in order to access the monitoring system tables.
+
+Additionally, as the metrics for current license usage use the values from the most recent [audit][audit command], it is recommended to schedule it to occur as often as possible. For more information, see [this][license guide].
+
+### Configuration
+
+1. Edit the `vertica.d/conf.yaml` file, in the `conf.d/` folder at the root of your Agent's configuration directory to start collecting your vertica performance data. See the example [vertica.d/conf.yaml][] for all available configuration options.
+
+2. [Restart the Agent][agent restart].
+
+#### Log Collection
+
+**Available for Agent >6.0**
+
+1. Collecting logs is disabled by default in the Datadog Agent, enable it in your `datadog.yaml` file:
+
+```
+logs_enabled: true
+```
+
+2. Add this configuration block to your `vertica.d/conf.yaml` file to start collecting your Vertica logs:
+
+```
+logs:
+  - source: vertica
+    type: file
+    path: /catalog-path/database-name/node-name_catalog/vertica.log
```

Comment:
Are those Placeholders ? if so the path should be: `/<CATALOG_PATH>/<DATABASE_NAME>/<NODE_NAME_CATALOG>/vertica.log`